### ISSUE TITLE
Omen 15-en1xxx going into sleep mode or shutdown prompt suddenly even if in use

### ISSUE THREAD

**Issue by mertcdev** [OmenMon/OmenMon#43]
Temps were fine but HwInfo and Remote System Monitor(mobile app to tracking temps) were open. In your answer in another issue, you stated that PC system going to sleep suddenly due to overloading the EC. 

I was experiencing a similar problem with the Notebook Fan Control application. Even though the laptop was fully charged, it was going to sleep mode after the battery level was below 5 percent message.

I don't want to go back OGH because it reduced to 35W cpu power limit on most games.

**Comment by GeographicCone** [comment]
Yes, these are the symptoms of EC overload. Workarounds include disabling the EC use by other applications and/or setting **OmenMon** to limit EC use. Here are some details: https://github.com/OmenMon/OmenMon/issues/35#issuecomment-1932079136

For a while now it's been possible to configure **OmenMon** for no background EC use at all, as long as you can accept being limited to the `BIOS` temperature sensor only: https://omenmon.github.io/config#sensors
